docs/standards/universal-profile/04-lsp6-key-manager.md

```diff
@@ -98,38 +100,76 @@ To restrict an `<address>` to only talk to a specific contract at address `<targ
 
 <br/>
 
-### <a name="allowed-functions"></a> Allowed functions
+### Allowed functions
 
 You can also restrict which functions a specific address can run, by providing a list of `bytes4` function selector.
 
 To restrict an `<address>` to only execute the function `transfer(address,uint256)` (selector: `a9059cbb`), the following key-value pair can be set in the ERC725Y contract storage.
 
 - **key:** `0x4b80742d000000008efe0000<address>`
-- **value:** `0xa9059cbb`
+- **value(s):** `[ 0xa9059cbb ]`
+
+The list (= array) of allowed `bytes4` function selectors **MUST be abi-encoded** (See the section [_Contract ABI Specification > Strict Encoding Mode_](https://docs.soliditylang.org/en/v0.8.11/abi-spec.html#strict-encoding-mode)) in the [Solidity documentation](https://docs.soliditylang.org/en/v0.8.11/abi-spec.html#).
 
 :::info Infos
 
 **If no bytes4 selectors are set, the caller address can execute any functions.**
 
 :::
 
+### Allowed standards
+
```

Comment:
I think adding a note that this type of permission doesn't offer security over the inner workings of a contract, therefore it's more of a "mistake prevention" rather than a security.